tests/de/gurkenlabs/litiengine/GameRandomTests.java

```diff
@@ -53,4 +111,108 @@ public void chooseStringArray_ThrowsOnEmpty() {
         // act, assert
         assertThrows(IllegalArgumentException.class, () -> gameRandom.choose(arr));
     }
+
+    @Test
+    public void chooseIntegerCollection_null(){
+        // arrange
+        Collection<Integer> collection = null;
+
+        // act
+        Integer result = this.gameRandom.choose(collection);
+
+        // assert
+        assertNull(result);
+    }
+
+    @Test
+    public void chooseIntegerCollection_empty(){
+        // arrange
+        Collection<Integer> collection = List.of();
```

Comment:
List.of() does not exist in Java 8.

Reply:
@Gamebuster19901 I replaced the List.of() call and checked for other usages (was the only one), should be fixed now.